README.markdown

```diff
@@ -237,6 +237,10 @@ Here is an example:
         ngx.say("failed to query: ", err)
         return
     end
+    if ans.errcode then
+        ngx.say("server returned the error state: ", ans.errcode, ans.errstr)
```

Comment:
This does not seem correct; `errcode` and `errstr` will be concatenated together without spacing or anything helping to distinguish them.